stable/ambassador/README.md

```diff
@@ -0,0 +1,164 @@
+| `adminService.nodePort`            | If explicit NodePort for admin service is required                              | `true`                        |
+| `adminService.type`                | Ambassador's admin service type to be used                                      | `ClusterIP`                   |
+| `ambassador.id`                    | Set the identifier of the Ambassador instance                                   | `default`                     |
+| `daemonSet`                        | If `true`, Create a daemonSet. By default Deployment controller will be created | `false`                       |
+| `env`                              | Any additional environment variables for ambassador pods                        | `{}`                          |
+| `image.pullPolicy`                 | Image pull policy                                                               | `IfNotPresent`                |
+| `image.repository`                 | Image                                                                           | `quay.io/datawire/ambassador` |
+| `image.tag`                        | Image tag                                                                       | `0.50.1`                      |
+| `imagePullSecrets`                 | Image pull secrets                                                              | `[]`                          |
+| `namespace.name`                   | Set the `AMBASSADOR_NAMESPACE` environment variable                             | `metadata.namespace`          |
+| `podAnnotations`                   | Additional annotations for ambassador pods                                      | `{}`                          |
+| `prometheusExporter.enabled`       | Prometheus exporter side-car enabled                                            | `false`                       |
+| `prometheusExporter.pullPolicy`    | Image pull policy                                                               | `IfNotPresent`                |
+| `prometheusExporter.repository`    | Prometheus exporter image                                                       | `prom/statsd-exporter`        |
+| `prometheusExporter.tag`           | Prometheus exporter image                                                       | `v0.8.1`                      |
+| `rbac.create`                      | If `true`, create and use RBAC resources                                        | `true`                        |
+| `replicaCount`                     | Number of Ambassador replicas                                                   | `1`                           |
+| `resources`                        | CPU/memory resource requests/limits                                             | `{}`                          |
+| `securityContext.user`             | If `true`, create a service for Ambassador's admin UI                           | `8888`                        |
```

Comment:
Can you put the whole securityContext in the values.yaml so users can set other child properties?